src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+import duke.Task;
+import duke.Deadline;
+import duke.Todo;
+import duke.Event;
+import duke.DukeException;
+import java.util.ArrayList;
+import java.io.*;
```

Comment:
Avoid wildcard imports! You may import the specific classes you require from the package. [See here](https://se-education.org/guides/conventions/java/basic.html#package-and-import-statements)